### ISSUE TITLE
Check accessibility

### ISSUE THREAD

**Issue by ecarriou** [system-sdk/system-designer#16]
Check if accessibility is ok, if not, fix it.

**Comment by therckenrath** [comment]
One missing accessibility feature is control over the size of the fonts used throughout the UI.
I have macular degeneration in both eyes which makes reading small fonts very difficult.

**Comment by ecarriou** [comment]
Thank you for your feedback !

I have added it to the list. I will check if 3rd party tools I use manage that.
A temporary solution would be maybe to use the web version of System Designer with integrated zoom feature of browsers. Only downside is that "Command +" / "Control +" shortcuts are overriden to ease items creation with keyboard.

**Comment by therckenrath** [comment]
Thank you for acknowledging my request 😀

I'm looking forward to see System Designer support this, in the meantime I
can make do with Android's full screen magnification.

I thoroughly enjoy using System Designer, thank you for creating it 🙏

Tjerk

**Comment by therckenrath** [comment]
Hi Erwan,

I ran into an issue and noticed that I was not using the latest version.

I renamed the existing installation (5.0.2) and then installed the latest
(5.1.0).

I now get the following error:
sh: 1: electron: not found

Then I renamed the latest installation and restored the previous
installation and now I get the same error with the previous installation.

I'm dead in the water...

Please, where did I go wrong and how do I fix it?

Tjerk

**Comment by ecarriou** [comment]
Can you tell me what did you do to install System Designer ? Is it an update from the Mac App Store ? And can you tell me what is your environment (macOS with intel / M1 chip) ?

I will try to reproduce your issue. But what you can do is delete the app and reinstall it.

**Comment by therckenrath** [comment]
I installed using the Build instructions on GitHub, in Crostini (Linux)
under Chrome OS, ie. git clone, followed by npm i

**Comment by ecarriou** [comment]
You clone this repo or [https://github.com/design-first/system-designer-electron](https://github.com/design-first/system-designer-electron) ? 

So you just clone the repo, do `npm i` and then `npm run start` ?

Try `npm ci` instead `npm I` to start from a clean base.

**Comment by therckenrath** [comment]
I tried again using npm ci and this time it worked.

I ran npm ci in system-designer, system-designer-electron and
system-runtime.

In my previous attempt I only ran npm i in the system-designer folder.

So it's all good now, thank you for your help.

I have confirmed that the issue I ran into on 5.0.2 still exists in 5.1.0 -
I'll create a bug report.

Tjerk

**Comment by therckenrath** [comment]
See issue #143
